src/components/Contentful/Grid.js

```diff
@@ -32,7 +32,7 @@ function GridContainer({
   if (layout === 'fluid') containerClassName += '-fluid'
 
   return (
-    <div className={className} id={id} linkTitle={linkTitle}>
+    <div className={className} id={id} linktitle={linkTitle}>
```

Comment:
I think the JSX convention is to have HTML attributes in camelcase.

```suggestion
    <div className={className} id={id} linkTitle={linkTitle}>
```

Reply:
This change I made as whenever I ran `npm run dev` it would throw a warning about it being in camel case, made the change to get rid of build warning. I'll double check to see if the change has effected anything